Makefile

```diff
@@ -578,29 +350,28 @@ po: $(UI_BUILD_FLAG_FILE)
 
 LANG = "en-us"
 ## generate API django .pot .po
-messages:
+messages: ## messages
 	@if [ "$(VENV_BASE)" ]; then \
 		. $(VENV_BASE)/awx/bin/activate; \
 	fi; \
 	$(PYTHON) manage.py makemessages -l $(LANG) --keep-pot
 
+## ¯\_(ツ)_/¯ dont know what this does
```

Comment:
Here's what it does!

    $ PYTHON=python3 make print-GIT_BRANCH print-COMPOSE_TAG
    feature/complex_extra_vars
    feature_complex_extra_vars

It lets you view variables in the context of the Makefile, neat!